benchmarking/repo_driver.py

```diff
@@ -360,6 +361,11 @@ def _getCommand(self, repo_info):
             " --framework " + getString(getArgs().framework) + \
             " --info " + info + " " + \
             ' '.join([getString(u) for u in unknowns])
```

Comment:
Is there a case that the env cannot be handled by the unknown arguments but need to be specialized?

Reply:
There is some differences for k-v args. Since the output format is like "--env a=1 b=2 c=3". I remember there is some issue when treating it as unknown args.